### DISCUSSION TITLE
CircleLayer to draw a big circle around a given point

### DISCUSSION THREAD

**Discussion by Hemkesh** [xclud/flutter_map#52] [Ideas]
Hi! I absolutely LOVE your package, and I was wondering if there is a way we can make a CircleLayer.

It would have a radius, center and color. This would make a circle around a canter point on the map.

**Comment by xclud** [comment]
Hi. This is not built in the package. But you can easily do it.

Look at the Markers sample. Instead of showing an `Icon`, show a `Container` with desired color and dimentions. This will look like a rectangle/square. Then set a shape for the Container to make it a circle/oval.

**Reply by Hemkesh** [reply]
Thank you for replying! I tried that but the circle was scaling with zoom, maybe I am just dumb and don't know how to stop that from happening.
